src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/ResourceServerConfiguration.kt

```diff
@@ -24,7 +24,7 @@ class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
           "/v3/api-docs/**", "/api/swagger.json", "/swagger-ui/**",
           "/v3/api-docs", "/swagger-ui.html",
           "/swagger-resources", "/swagger-resources/configuration/ui", "/swagger-resources/configuration/security",
-          "/queue-admin/retry-all-dlqs",
+          "/queue-admin/retry-all-dlqs", "/process-past-date-restricted-patients",
```

Comment:
don't want to create an access token to call the endpoint so removed from security